tests/app/delivery/test_send_to_providers.py

```diff
@@ -724,51 +730,3 @@ def test_send_email_to_provider_uses_reply_to_from_notification(
         html_body=ANY,
         reply_to_address="test@test.com"
     )
-
-
-def test_send_email_to_provider_should_format_reply_to_email_address(
-        sample_email_template,
-        mocker):
-    mocker.patch('app.aws_ses_client.send_email', return_value='reference')
-
-    db_notification = create_notification(template=sample_email_template, reply_to_text="test@test.com\t")
-
-    send_to_providers.send_email_to_provider(
-        db_notification,
-    )
-
-    app.aws_ses_client.send_email.assert_called_once_with(
-        ANY,
-        ANY,
-        ANY,
-        body=ANY,
-        html_body=ANY,
-        reply_to_address="test@test.com"
-    )
-
-
-def test_send_sms_to_provider_should_format_phone_number(sample_notification, mocker):
-    sample_notification.to = '+44 (7123) 123-123'
-    send_mock = mocker.patch('app.mmg_client.send_sms')
-
-    send_to_providers.send_sms_to_provider(sample_notification)
-
-    assert send_mock.call_args[1]['to'] == '447123123123'
-
-
-def test_send_email_to_provider_should_format_email_address(sample_email_notification, mocker):
```

Comment:
Optional: You could keep this test and the one above it but rejig them so it's

`test_send_email_to_provider_uses_normalised_to_address_rather_than_email_address`

which would persist a notification with one value for the `to` field and a different value for the `normalised_to` field and then checks that the we call `send_mock` with the normalised value and not the unnormalised value.

You have this behaviour potentially covered in some of your tests above but the testing of this functionality above is all implicit, a separate test would make clear the behaviour that we expect